### PR TITLE
Issue 1393: Missing panel's gap

### LINKED ISSUE #1393: Missing panel's gap
**When Safetibase panel is opened via a risk pin and there is already the Issues panel open, the panel spacing is not there**

Caused by #1273 #1343 

feb-21-2019 16-06-27

_Originally posted by @Charence in https://github.com/3drepo/3drepo.io/issues/1391#issuecomment-466072154_

### PR DESCRIPTION
**Pull request by simplypixi**
This fixes #1393

### CODE REVIEW

File: frontend/components/panel/js/panel.component.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { sumBy, clamp } from 'lodash';
+import { sumBy, clamp, groupBy } from 'lodash';
```
Comment (sanmont3drepo) [inline frontend/components/panel/js/panel.component.ts]: Unused groupBy  😁

Comment (simplypixi) [reply frontend/components/panel/js/panel.component.ts]: Ugh... xd

Comment (simplypixi) [reply frontend/components/panel/js/panel.component.ts]: Fixed ;)

**Comment by simplypixi** [comment]
@sanmont3drepo Fixed ;)

**Pull request merged**